web/themes/custom/vote_gov/templates/block/block-content--basic.html.twig

```diff
@@ -2,23 +2,22 @@
 
 {% block content %}
 {% set classes = [
+    'bg-lightblue',
```

Comment:
We don't want to set this in all basic blocks. You can remove. We can add it to the classes field in the block